### DISCUSSION TITLE
Seems like the newest Pinecil batch is indeed using the SC7A20 accelerometer

### DISCUSSION THREAD

**Discussion by strayer** [Ralim/IronOS#908] [General]
So I noticed that the latest release from January includes support for the SC7A20 accelerometer because "its possible the Pinecil will move to this accelerometer in a future batch".

Not sure if it is of any help, but the Pinecil I purchased on 2021-03-12 seems to have that, according to the PCB info on the debug menu. Looks like they actually did move to that accelerometer. Sorry for the noise if this was already known!

**Comment by Ralim** [comment]
I appreciate the confirmation. I was fairly sure this would be the case as the SC7A20 has more stock around but wasn't 100% on this.
I was sent a sample unit with that accelerometer to get it working, so it should all work (but as always; give a yell if something breaks).

Thanks for the heads up :)
